### PR TITLE
Fix reverse option 

### LINKED ISSUE #473: reverse option does not work
It looks like that reverse option does not work at all. Or maybe I got wrong purpose of that feature. I thought that if reverse is set to true then checked state is pin on left and uncheck is right. But reverse set to true did not do that. It could be replicated even on demo and Stackblitz sample project.

### PR DESCRIPTION
**Pull request by blazekv**
fixes #473

I am not sure if this is best way how to fix it, but there were two main issues here.

On was that reverse option did not switch labels (which I fixed by setting CSS  class 'switch-label-checked' and 'switch-label-unchecked' based on reverse.

Second one was that in case of default behavior color and defaultBgColor was not set and therefore line 135
` return !this.checked ? this.color : this.defaultBgColor;` did not have effect (color was took from CSS).

### CODE REVIEW

File: src/lib/ui-switch/ui-switch.component.html [hunk 1]
```diff
@@ -16,13 +16,13 @@
   <label class="switch-pane" *ngIf="checkedLabel || uncheckedLabel">
     <span
       [attr.aria-label]="this.checkedLabel"
-      class="switch-label-checked"
```
Comment (cmckni3) [inline src/lib/ui-switch/ui-switch.component.html:19]: Maybe use a method with the logic?

File: src/lib/ui-switch/ui-switch.component.html [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
+      [class]="reverse ? 'switch-label-unchecked' : 'switch-label-checked'"
       [style.color]="getColor('checkedTextColor')"
       >{{ this.checkedLabel }}</span
     >
     <span
       [attr.aria-label]="this.uncheckedLabel"
-      class="switch-label-unchecked"
```
Comment (cmckni3) [inline src/lib/ui-switch/ui-switch.component.html:25]: Maybe use a method with the logic?

File: src/lib/ui-switch/ui-switch.component.ts [hunk 3]
```diff
@@ -104,10 +104,10 @@ export class UiSwitchComponent implements ControlValueAccessor, OnDestroy {
     private cdr: ChangeDetectorRef
   ) {
     this.size = (config && config.size) || 'medium';
-    this.color = config && config.color;
```
Comment (cmckni3) [inline src/lib/ui-switch/ui-switch.component.ts:107]: Remove default

File: src/lib/ui-switch/ui-switch.component.ts [hunk 4]
```diff
@@ -104,10 +104,10 @@ export class UiSwitchComponent implements ControlValueAccessor, OnDestroy {
...
+    this.color = (config && config.color) || 'rgb(100, 189, 99)';
     this.switchOffColor = config && config.switchOffColor;
     this.switchColor = config && config.switchColor;
-    this.defaultBgColor = config && config.defaultBgColor;
```
Comment (cmckni3) [inline src/lib/ui-switch/ui-switch.component.ts:110]: Remove default

**Review by cmckni3 — CHANGES_REQUESTED** [review]
See comments

File: src/lib/ui-switch/ui-switch.component.html (hunk 1 above)
Comment (blazekv) [reply src/lib/ui-switch/ui-switch.component.html:19]: OK, i move it into method

**Review by cmckni3 — APPROVED** [review]

**Comment by cmckni3** [comment]
Thanks!

**Pull request merged**